diagnostic.md

```diff
@@ -7,40 +7,51 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+https://github.com/donpowers/git-diagnostic
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+you clone it
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git status
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git add diagnostic.md
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+The first line should be a like a subject line in an email..."Fixed bug in
+the howManyDays()"
+
+Use bullet type items to explain in more detail what was changed.
+
+ - fixed array index problem when incorrect day was given
 
 -7.Should you ever edit published history?
 
- <!-- Remove this comment and place your answer here. -->
+ No. I think this would mess things up. Not sure what a better practice should be
+
+found this on the web...
+However, keep two important details in mind:
+
+Amend only works with the very last commit. If you notice your mistake only after adding another commit, amend won't help you much.
+Amend rewrites the commit history in your repository: the old commit is replaced by a completely new one (a new and different commit object). This makes it very important that you don't amend (= rewrite) commits that you've already published to a remote repository! Because in that case, your colleagues might have already based their work on this commit - which you would try to replace using "amend".
```

Comment:
Great research here Don. You nailed this one. Editing published history can be VERY dangerous because if multiple people are working from the same code base you want to ensure that everyone's history matches exactly. Otherwise when you go to merge files you can encounter conflicts that can be extremely difficult to resolve.